### PR TITLE
Switch kubernetes-e2e-gce-enormous-cluster and kubernetes-e2e-gke-large-cluster testing to GCI image

### PR DESCRIPTION
**Pull request by mtaufen**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/657)

### CODE REVIEW

**Comment by vishh** [comment]
@wojtek-t @dchen1107 We are moving to GCI on both GKE and GCE k8s. I recommend switching large cluster tests to GCI completely since v1.5 will be running GCI. Any objections?

**Comment by wojtek-t** [comment]
LGTM

File: jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml [hunk 1]
```diff
@@ -162,6 +162,7 @@
                 export GKE_CREATE_FLAGS="--max-nodes-per-pool=1000"
                 export CLOUDSDK_CONTAINER_USE_CLIENT_CERTIFICATE=True
                 export CLOUDSDK_API_ENDPOINT_OVERRIDES_CONTAINER="https://staging-container.sandbox.googleapis.com/"
+                export KUBE_GKE_IMAGE_TYPE="gci"
```
Comment (wojtek-t) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml]: How is this related to:
https://github.com/kubernetes/test-infra/pull/664/files#diff-3945c589c3c116b9fd9824f0783f1905R184

Comment (mtaufen) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml]: This is a GKE test, `export KUBE_GKE_IMAGE_TYPE="gci"` is correct. The reason to just move these to gci rather than cloning is that these tests require enormous quota, and it is not feasible to get quota for a clone at this time.
